package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "test-backend",
+  "description": "Wiredcraft backend test for Oliver.Wang(ole3021)",
+  "version": "1.0.0",
+  "main": "server/server.js",
+  "scripts": {
+    "lint": "standard server/**/*.js",
+    "server": "nodemon . --exec babel-node --config ./configs/nodemon.json",
+    "server-test": "NODE_ENV=test babel-node .",
+    "test": "npm run posttest && ava -verbose",
+    "posttest": "npm run lint && nsp check"
+  },
+  "dependencies": {
+    "babel-cli": "^6.24.0",
```

Comment:
yes, it should be in devDependencies, and not in dependencies. 

This is a mistake i make, i imagine to use the `babel-node` in production, so it will be required in dependencies, but this my fault. 

* the `babel-node` should be not used in production environment (not in dependencies), cause it is pretty heavy with high memory useage.
* the correct way to do is build a compiled version of the Babel code into ES5 code, and execute the server with `node`